frontend/src/error/errorHandler.ts

```diff
@@ -26,7 +26,7 @@ export const getErrorMessage = (err: ErrorResponse) => {
   if (err.message) {
     try {
       const errorJSON = JSON.parse(err.message);
-      return "error" in errorJSON ? errorJSON.error : "title" in errorJSON ? errorJSON.title : defaultErrorMessage;
```

Comment:
I've always stayed away from the key in object pattern in Javascript/Typescript because the or syntax here actually provides a nicer understanding of the default. This change also checks to make sure that the errorJSON was successfully parsed before trying to check for keys.

Reply:
Hi @williamtaggart97, thanks for this change, it actually makes the code more readable. This function is currently returning `string | undefined` because the `if` doens't have an `else` branch. Could you add `else return defaultErrorMessage;` ? Another very minor suggestion would be to add a new type `Error` in `global.d.ts`, something like: 

```typescript
interface Error {
  title: string;
}
```

Reply:
fixing now